File: tl/src/main/java/com/github/badoualy/telegram/tl/api/TLAbsBotInlineResult.java

package com.github.badoualy.telegram.tl.api;


import com.github.badoualy.telegram.tl.core.TLObject;



public abstract class TLAbsBotInlineResult extends TLObject {

    protected String id;

    protected String type;

    protected com.github.badoualy.telegram.tl.api.TLAbsBotInlineMessage sendMessage;


    public String getId() {
        return id;
    }

    public void setId(String value) {
        this.id = value;
    }

    public String getType() {
        return type;
    }

    public void setType(String value) {
        this.type = value;
    }

    public com.github.badoualy.telegram.tl.api.TLAbsBotInlineMessage getSendMessage() {
        return sendMessage;
    }

    public void setSendMessage(com.github.badoualy.telegram.tl.api.TLAbsBotInlineMessage value) {
        this.sendMessage = value;
    }

}

File: tl/src/main/kotlin/com/github/badoualy/telegram/tl/api/TLAbsBotInlineResult.kt
package com.github.badoualy.telegram.tl.api

import com.github.badoualy.telegram.tl.core.TLObject

/**
 * Abstraction level for the following constructors:
 * * [botInlineMediaResult#17db940b][TLBotInlineMediaResult]
 * * [botInlineResult#9bebaeb9][TLBotInlineResult]
 *
 * @author Yannick Badoual yann.badoual@gmail.com
 * @see <a href="http://github.com/badoualy/kotlogram">http://github.com/badoualy/kotlogram</a>
 */
abstract class TLAbsBotInlineResult : TLObject() {
    abstract var id: String

    abstract var type: String

    abstract var sendMessage: TLAbsBotInlineMessage
}
